### PR TITLE
Title Component

### LINKED ISSUE #9: Title Component
image

Props: text
The acm documentation will explain how you can import that image called "Landing.svg" from the public/svg folder. 
It will also explain how to make and use a prop, and how to use Next.js image.
This Title will appear in the url: localhost:3000/about

In order to do this, you will need to use flexbox and absolute positioning.
Write code in src/components/Title.jsx

### PR DESCRIPTION
**Pull request by brianhelewa**
Fixed Footer.jsx class to className because in JSX you need to write className instead of class. I needed to do this to display about page.

image

Screen shot of error in Footer.jsx that was fixed:
Screen Shot 2024-01-15 at 6 50 23 PM
Pasted Graphic

### CODE REVIEW

File: src/components/Title.jsx [hunk 1]
```diff
@@ -1,7 +1,23 @@
 import React from "react";
 
 const Title = () => {
-  return <div>Title</div>;
+  const containerHeight = "170px";
```
Comment (SafeDuck) [inline src/components/Title.jsx]: delete, not needed

File: src/components/Title.jsx [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+
+  return (
+    <div
+      className="relative w-2/3 overflow-hidden"
+      style={{ height: containerHeight }}
```
Comment (SafeDuck) [inline src/components/Title.jsx]: only use tailwind classes in the className. Don't use style tags for css

File: src/components/Title.jsx [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+    >
+      <img
```
Comment (SafeDuck) [inline src/components/Title.jsx]: Use next image, not html image. Checkout ACM Docs for more info

File: src/components/Title.jsx [hunk 4]
```diff
@@ -1,7 +1,23 @@
 import React from "react";
 
 const Title = () => {
```
Comment (SafeDuck) [inline src/components/Title.jsx]: Pass in a prop called "text"

File: src/components/Title.jsx [hunk 5]
```diff
@@ -1,7 +1,23 @@
...
+      className="relative w-2/3 overflow-hidden"
+      style={{ height: containerHeight }}
+    >
+      <img
+        src="/svg/Landing.svg"
+        alt="Landing"
+        className="absolute bottom-0 left-0 w-full h-auto"
+      />
+      <div className="absolute bottom-0 left-0 right-0 flex justify-center items-center">
+        <div>Title</div>
```
Comment (SafeDuck) [inline src/components/Title.jsx]: Display your text prop on line 17 `{text}`
Change font, font weight, font size, and text color to match the figma

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/Title.jsx (hunk 1 above)
Comment (brianhelewa) [reply src/components/Title.jsx]: Thanks I removed it

File: src/components/Title.jsx (hunk 5 above)
Comment (brianhelewa) [reply src/components/Title.jsx]: image
Not sure what font to use but this was similar

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**